Project.toml

```diff
@@ -32,7 +32,9 @@ NetCDF = "30363a11-5582-574a-97bb-aa9a979735b9"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+EnsembleKalmanProcesses = "aa8a2aa5-91d8-4396-bcef-d4f2ec43552d"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 
 [targets]
-examples = ["CairoMakie", "NetCDF", "Interpolations", "Statistics", "Printf", "JLD2", "DataDeps"]
+examples = ["CairoMakie", "NetCDF", "Interpolations", "Statistics", "Printf", "JLD2", "DataDeps", "EnsembleKalmanProcesses", "Distributions"]
```

Comment:
what does this line do? `examples = [...]`?

Reply:
Its meant to be so you can do `julia --project=examples` and its like `examples` has a Project.toml with the content of the dependencies in this Project.toml + these